packages/automl/frontend/src/app/topology/tree-view/TreeNode.scss

```
@@ -0,0 +1,47 @@
+  transform-origin: center;
+  transform-box: fill-box;
+}
+
+.automl-tree-node__label {
+  width: 100%;
+  text-align: center;
+  color: var(--pf-t--global--text--color--subtle);
+  font-size: var(--pf-t--global--font--size--body--sm);
+  font-family: var(--pf-t--global--font--family--body);
+  line-height: 1.3;
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)